### ISSUE TITLE
如何在par_iter中使用Commands

### ISSUE THREAD

**Issue by wzjsun** [bevyengine/bevy#8385]
I have a requirement that there are two existing components A and B

I need to iterate over the entity containing component A, create a corresponding entity for it, and insert component B. Note that I want to create a new entity instead of directly inserting component B onto the existing entity

I am doing this because I do not want to adjust the prototype. There are many other components on the original entity, and the cost of adjusting the prototype may be high. At the same time, I also want component B to have continuous storage space

```rust
pub fn (query: Query<(), With<A>>, commands: Commands) {
     query.iter().map(|| {
         commands.spawn(B);
    })
}
```

If there are too many queries, I hope the iteration can be parallel, but the following code cannot pass:

```rust
pub fn (query: Query<(), With<A>>, commands: Commands) {
     query.par_iter().map(|| {
         commands.spawn(B);
    })
}
```

是否有一个方法能够满足我的需要呢？

**Comment by mockersf** [comment]
You can use [`ParallelCommands`](https://docs.rs/bevy/latest/bevy/ecs/prelude/struct.ParallelCommands.html) 🙂

**Comment by wzjsun** [comment]
@mockersf 

Yes, thank you very much. I'm sorry for asking such a basic question
